cs4660/graph/graph.py

```diff
@@ -113,50 +170,123 @@ def __init__(self):
         self.nodes = []
 
     def adjacent(self, node_1, node_2):
-        pass
+        # check if the two nodes are adjacent
+        if self.adjacency_matrix[self.__get_node_index(node_1)][self.__get_node_index(node_2)] != 0:
+            return True
+        else:
+            return False
```

Comment:
You can simply return `self.adjacency_matrix[self.__get_node_index(node_1)][self.__get_node_index(node_2)] != 0`